client/extensions/woocommerce/state/ui/payments/methods/README.md

```diff
@@ -0,0 +1,58 @@
+Payment Methods
+================
+
+This module is used to manage payment method related UI.
+
+## Actions
+
+### `openPaymentMethodForEdit( siteId: number, id: string )`
```

Comment:
I'm cool with whatever scheme we've decided on, but maybe we can discuss this in a group chat sometime to make sure we all agree?